### DISCUSSION TITLE
Bind the Left Command to switch into language A, and Right Command to language B

### DISCUSSION THREAD

**Discussion by AmirSimantov** [pqrs-org/Karabiner-Elements#3590] [General]
Hi.

Celebrating my new Mac with Ventura OS installed!

Switching between two languages ("Input sources") can be a bit confusing sometimes because I'm never really sure which language I'm in without checking the menu icon. I wish there was a specific key combination I could use, regardless of the currently selected language. That way, I could just use it before typing to make sure I'm in the right language.

What I'd like to do is use the Left Command key for one language and the Right Command key for the other. Here are a couple of alternatives I've thought of:

- Double-clicking the Left/Right Command button.
- Using a combination of the Left/Right Command key and Space.

I'm wondering if this kind of customization is possible using Karabiner, but I couldn't find it in the documentation. Any ideas?

Thanks!

**Comment by tester-andre** [comment]
try to this json for Complex Modification
https://gist.github.com/piskov/d20169dfae397c05b46e33d4042b0dbb#file-cmd-to-birman-json
![0896400c536485e9340256028ddcececd4ff35f326f4193e1a530b24ec35bf07](https://github.com/pqrs-org/Karabiner-Elements/assets/141130450/e6a39db6-e9f5-4ee5-a035-4c82b4759f0b)
